docs/index.md

```diff
@@ -9,13 +9,16 @@ Documentation
  - [Installation](./installation.md)
  - [CLI reference](./cli.md)
 
+### Advanced
+ - [Architecture](./architecture.md)
+ - [Security](./security.md)
+ - [CSI Specification](./specification.md)
+
```

Comment:
Can we also mention the minimum version requirements? Like, Kubernetes version, sidecar container versions [csi-node-driver-registrar, csi-provisioner, csi-driver, etc]

Reply:
Good point

Reply:
I've added it to installation docs